pkg/storaged/client.js

```diff
@@ -694,29 +744,16 @@ function nfs_mounts() {
         return spawn_nfs_mounts(["unmount", JSON.stringify(entry)]);
     }
 
-    function pids_and_units(users) {
-        return [
-            users.filter(u => u.pid).map(u => u.pid),
-            users.filter(u => u.unit).map(u => u.unit)
-        ];
-    }
-
     function stop_and_unmount_entry(users, entry) {
-        const [pids, units] = pids_and_units(users);
-        return spawn_nfs_mounts(["stop-and-unmount",
-            JSON.stringify(units), JSON.stringify(pids),
-            JSON.stringify(entry)]);
+        return client.stop_mount_users(users).then(() => unmount_entry(entry));
     }
 
     function stop_and_remove_entry(users, entry) {
-        const [pids, units] = pids_and_units(users);
-        return spawn_nfs_mounts(["stop-and-remove",
-            JSON.stringify(units), JSON.stringify(pids),
-            JSON.stringify(entry)]);
+        return client.stop_mount_users(users).then(() => remove_entry(entry));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16991-20220825-111327-558828a7-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)